jane/doc/extensions/uniqueness/intro.md

```diff
@@ -0,0 +1,123 @@
+# Introduction to Uniqueness
+
+See also the full feature [reference](reference.md) and [common pitfalls](pitfalls.md).
+In this document, we use the new [syntax for modes](../modes/syntax.md),
+which we expect to be stable soonish.
+
+The `unique` mode designates values that have only a single reference pointing
+to them. For example, an externally allocated data structure might support a
+`free` operation:
+
+```ocaml
+val free : t @ unique -> unit
+```
+
+The compiler guarantees that this operation will be sound: as the argument is
```

Comment:
Hmm is the operation unsound prior to uniqueness? I guess "sound" is a strong and vague word. Maybe we can just say "With the unique annotation, the compiler can now prevents use-after-free"?

Reply:
Right, "sound" is too PLy. I'll remove it